docs/FAQ.md

```diff
@@ -373,3 +373,38 @@ stripping out nested `vendor` directories.
 ## How do I configure a dependency that doesn't tag its releases?
 
 Add a constraint to `Gopkg.toml` that specifies `branch: "master"` (or whichever branch you need) in the `[[constraint]]` for that dependency. `dep ensure` will determine the current revision of your dependency's master branch, and place it in `Gopkg.lock` for you. See also: [What is the difference between Gopkg.toml and Gopkg.lock?](#what-is-the-difference-between-gopkgtoml-the-manifest-and-gopkglock-the-lock)
+
+## How do I use `dep` with Docker?
+
+`dep ensure -vendor-only` creates the vendor folder from a valid `Gopkg.toml` and `Gopkg.lock` without checking for Go code.
+This is especially useful for builds inside docker utilizing cache layers.
+
+Sample dockerfile:
+
+    FROM golang:1.8 AS builder
+
+    RUN curl -fsSL -o /usr/local/bin/dep https://github.com/golang/dep/releases/download/vX.X.X/dep-linux-amd64 && chmod +x /usr/local/bin/dep
```

Comment:
Hi, just came over the docker file issue. Might wanna add 
```
RUN curl -fsSL -o /tmp/dep.zip \
    https://github.com/golang/dep/releases/download/v0.3.0/dep-linux-amd64.zip \
    && unzip /tmp/dep.zip -d /usr/local/bin \
    && chmod +x /usr/local/bin/dep 
```

Reply:
This PR is waiting on our new release scheme, which won't use zipped binaries on our GitHub releases.